doc/rbd/rbd-config-ref.rst

```diff
@@ -146,45 +146,90 @@ RBD supports advanced features which can be specified via the command line when
 :CLI value: layering
+:Added in: v0.70 (Emperor)
+:KRBD support: since v3.10
+:Default: yes
 
 ``Striping v2``
 
 :Description: Striping spreads data across multiple objects. Striping helps with parallelism for sequential read/write workloads.
 :Internal value: 2
 :CLI value: striping
+:Added in: v0.70 (Emperor)
+:KRBD support: since v3.10
+:Default: yes
 
 ``Exclusive locking``
 
 :Description: When enabled, it requires a client to get a lock on an object before making a write. Exclusive lock should only be enabled when a single client is accessing an image at the same time. 
 :Internal value: 4
 :CLI value: exclusive-lock
+:Added in: v0.92 (Hammer)
+:KRBD support: since v4.9
+:Default: yes
 
 ``Object map``
 
 :Description: Object map support depends on exclusive lock support. Block devices are thin provisioned—meaning, they only store data that actually exists. Object map support helps track which objects actually exist (have data stored on a drive). Enabling object map support speeds up I/O operations for cloning; importing and exporting a sparsely populated image; and deleting.
 :Internal value: 8
 :CLI value: object-map
+:Added in: v0.93 (Hammer)
+:KRBD support: no
+:Default: yes
 
 
 ``Fast-diff``
 
 :Description: Fast-diff support depends on object map support and exclusive lock support. It adds another property to the object map, which makes it much faster to generate diffs between snapshots of an image, and the actual data usage of a snapshot much faster.
 :Internal value: 16
 :CLI value: fast-diff
+:Added in: v9.0.1 (Infernalis)
+:KRBD support: no
+:Default: yes
 
 
 ``Deep-flatten``
 
 :Description: Deep-flatten makes rbd flatten work on all the snapshots of an image, in addition to the image itself. Without it, snapshots of an image will still rely on the parent, so the parent will not be delete-able until the snapshots are deleted. Deep-flatten makes a parent independent of its clones, even if they have snapshots.
 :Internal value: 32
 :CLI value: deep-flatten
+:Added in: v9.0.2 (Infernalis)
+:KRBD support: no
+:Default: yes
 
 
 ``Journaling``
 
 :Description: Journaling support depends on exclusive lock support. Journaling records all modifications to an image in the order they occur. RBD mirroring utilizes the journal to replicate a crash consistent image to a remote cluster.
 :Internal value: 64
 :CLI value: journaling
+:Added in: v10.0.1 (Jewel)
+:KRBD support: no
+:Default: no
+
+
+``Data pool``
+
+:Description: On erasure-coded pools, the image data block objects need to be stored on a separate pool from the image metadata.
+:Internal value: 128
+:Added in: v11.1.0 (Kraken)
+:KRBD support: since v4.11
+:Default: no
+
+
+``Operations``
+
+:Description: Used to restrict older clients from performing certain maintenance operations against an image (e.g. clone, snap create).
+:Internal value: 256
+:Added in: v13.0.2 (Mimic)
+:KRBD support: since v4.16
+
+
+``Migrating``
+
+:Internal value: 512
+:Added in: v14.0.1
```

Comment:
Nit: ```v14.0.1 (Nautilus)```

Reply:
And for "Description" you could have something like this:

Used to restrict older clients from opening an image when it is in migration state.

Reply:
Thanks!